src/13-serial-over-bluetooth/README.md

```diff
@@ -22,4 +22,6 @@ Recommended steps to wire this up:
 And that's it! You should be able to run all the programs you wrote in [section 11] without
 modification! Just make sure you open the right serial device / COM port.
 
+**NOTE** If you are having trouble communicating with the bluetooth device, you may need to initialize USART1 with a lower baud rate. Lowering it from 115,200 bps to 9,600 bps might help, as described [here](https://github.com/rust-embedded/discovery/blob/master/src/11-usart/auxiliary/src/lib.rs#L31)
+
```

Comment:
I was stuck on this step, and after some debugging, I discovered that the program was getting stuck on this line:

```rust
while usart1.isr.read().rxne().bit_is_clear() {}
```

After some exploration - and some help from a friend! - I discovered that it was due to the baud mismatch between the chip and the bluetooth module.  Setting the baud rate to `9600.bps()`, as mentioned above, fixed the issue.